packages/calcite-components/src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -83,7 +83,7 @@ export class ColorPickerHexInput implements LoadableComponent {
   //--------------------------------------------------------------------------
 
   /**
-   * When `true`, an empty color (`null`) will be allowed as a `value`.
+   * When `true`, an empty color (`undefined`) will be allowed as a `value`.
```

Comment:
This entire component is internal, so I am assuming the change from `null` to `undefined` is not a breaking change?